rollup.config.js

```diff
@@ -0,0 +1,55 @@
+const rollup = require('rollup').rollup;
```

Comment:
We don't need non-minified versions, so you can produce minified versions right here